### PR TITLE
Remove Rebass from the project

### LINKED ISSUE #155: Create Show Form Throws Errors w/ Datepicker
For some reason, there are errors thrown which prevent the page from rendering. We currently have the component commented out.

1. Find why the errors are being thrown
2. Fix it so that we can render the form

### PR DESCRIPTION
**Pull request by robmcl4**
Removes Rebass.

Also restructures how the home page scales, hiding the accent image on `xs` screens and fixing the ever-present horizontal scrollbar.

Also tweaks how buttons look on the Show card (sorry, couldn't help myself)

Album of changes: https://imgur.com/a/2vd1w 

Closes #155
Closes #130 (Duplicate PR)

### CODE REVIEW

File: frontend/src/Home/Page.js [hunk 1]
```diff
@@ -23,6 +23,13 @@ const ButtonContainer = styled.div`
   width: 250px;
 `
 
+const LoginContainer = styled.div`
+  justify-content: center;
+  min-height: 75vh;
+  display: flex;
+  flex-direction: column;
```
Comment (chtinahow) [inline frontend/src/Home/Page.js]: This will definitively give you the vertical layout you need for the login area, but it's not horizontally centered correctly. You'll probably want to add a separate div or something else around the login container to center it.

Comment (chtinahow) [reply frontend/src/Home/Page.js]: I might just be pedantic about it, so don't work super hard on it if you can't get it centered

File: frontend/src/Home/Page.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-import { Flex } from 'rebass'
+import { Container, Row, Col } from 'reactstrap'
 import styled from 'styled-components'
 
 import splash from 'assets/splash-page.jpg'
 
 const SplashImage = styled.img`
   background-image: url(${props => props.src});
-  background-size: cover;
+  object-fit: cover;
```
Comment (billdybas) [inline frontend/src/Home/Page.js]: Not supported in IE 11

File: frontend/src/index.js [hunk 3]
```diff
@@ -1,3 +1,10 @@
+// Initialize Global Imports
```
Comment (billdybas) [inline frontend/src/index.js]: Why did you move these?

Comment (robmcl4) [reply frontend/src/index.js]: fixes things

File: frontend/src/Home/Page.js (hunk 1 above)
Comment (robmcl4) [reply frontend/src/Home/Page.js]: fixed centering in future commit

**Comment by billdybas** [comment]
LGTM. 🚀 

Only qualm (and technically a non-requirement) I have is on `CreateShowForm` I would move the date pickers underneath the other form fields on smaller screens – right now they collapse horizontally. I can do this when I get to #156.
